src/translations/main_nl.ts

```diff
@@ -703,6 +705,7 @@ Het logboek is opgeslagen als %2</translation>
         <source>Snooze for ...</source>
         <translation>Onderdrukken:</translation>
     </message>
+        <translatorcomment>checkTranslation ignore: punctuation_end_differ</translatorcomment>
```

Comment:
```suggestion
        <translation>Onderdrukken:</translation>
        <translatorcomment>checkTranslation ignore: punctuation_end_differ</translatorcomment>
    </message>
```
Unfortunately, this `translatorcomment` is in the wrong spot.